services/ftp/cmd.go

```diff
@@ -671,6 +696,12 @@ func (cmd commandRest) RequireAuth() bool {
 }
 
 func (cmd commandRest) Execute(conn *Conn, param string) {
+
+	if conn.user == "anonymous" {
```

Comment:
can we use `IsAnonymous(conn.User)` here? So we can have "anonymous" hardcoded at a single place?